src/modules/shared/components/MediaCard/MediaCard.tsx

```diff
@@ -120,6 +120,7 @@ const MediaCard: FC<MediaCardProps> = ({
 				return renderImage(preview ? '/images/waveform.svg' : undefined);
 
 			case 'video':
+				return renderImage(preview);
 			case 'film':
 				return renderImage(preview);
```

Comment:
Deze regel verandert niets aan de code. Dit wordt een fall-through case genoemd voor een switch statement: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/switch#breaking_and_fall-through

Doordat case video geen break of return heeft, zal die gewoon verder executen in de body van de film. Zo wordt er voor zowel video als film dezelfde code uitgevoerd.
```
			case 'video':
			case 'film':
				return renderImage(preview);
```